components/extensions/org.wso2.carbon.ml.siddhi.extension/src/main/java/org/wso2/carbon/ml/siddhi/extension/PredictStreamProcessor.java

```diff
@@ -132,23 +202,24 @@ public void start() {
         try {
             populateFeatureAttributeMapping();
         } catch (Exception e) {
```

Comment:
Fixed in the PR: https://github.com/wso2/carbon-ml/pull/135
to catch specific exception.